source/releasenotes/2024-06-04-pantheon-mu-plugin-1-4-4-wordpress-composer-managed-update.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Pantheon MU Plugin v1.4.4 and WordPress (Composer Managed) update
+published_date: "2024-06-04"
+categories: [wordpress, plugins]
+---
+
+The latest [1.4.4 update](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin is now available. This update fixes the WordPress Multisite network setup process by by ensuring the correct instructions for WordPress Multisite setup on Pantheon environments are always shown. This update will be included with the next WordPress release. Composer-based WordPress installs can get the update today by running `composer update`.
+
+The latest [WordPress (Composer Managed) (Early Access)](https://github.com/pantheon-upstreams/wordpress-composer-managed) release updates the configuration file filename displayed in the network setup screen to specify `config/application.php` to apply your configuration changes to, rather than the default `wp-config.php` for non-Bedrock Composer installs. This update will be pushed out to all WordPress (Composer Managed)-based installs alongside the MU Plugin update.
```

Comment:
@jazzsequence what's the relationship between these two changes? Does the WordPress Composer Manage change also relate to Multisite?

If these are actually to unrelated changes that are just happening on the same day, I think we should do two separate release notes.

Reply:
Yes, they relate to each other in that the page receiving the updates is _in the mu-plugin_ but the instructions need to be differentiated _in the Composer Managed_ upstream. I can make that clearer.